vspace/src/main/java/edu/asu/diging/vspace/core/services/IModuleManager.java

```diff
@@ -1,11 +1,24 @@
 package edu.asu.diging.vspace.core.services;
 
+import java.util.List;
+
+import edu.asu.diging.vspace.core.exception.ModuleDoesNotExistException;
 import edu.asu.diging.vspace.core.model.IModule;
+import edu.asu.diging.vspace.core.model.ISlide;
+import edu.asu.diging.vspace.core.services.impl.CreationReturnValue;
 
 public interface IModuleManager {
 
     IModule storeModule(IModule module);
 
     IModule getModule(String id);
+    
+    ISlide createSlide(String id, String title, String description) throws ModuleDoesNotExistException;
+
+    CreationReturnValue storeSlide(ISlide space, byte[] image, String filename);
```

Comment:
why is the slide parameter called 'space'?

Reply:
a slide can have multiple images, so it doesn't seem to make sense to pass only one here.